src/app/component/shared/form/form.component.ts

```diff
@@ -26,69 +23,38 @@ import { ApiResponse } from "src/app/services/baw-api/baw-api.service";
   encapsulation: ViewEncapsulation.None
 })
 export class FormComponent extends WithUnsubscribe() implements OnInit {
-  @Input() btnColor:
-    | "btn-danger"
-    | "btn-success"
-    | "btn-warning"
-    | "btn-primary"
-    | "btn-secondary"
-    | "btn-info" = "btn-success";
   /**
-   * Deprecated
+   * DEPRECATED
```

Comment:
the diff is a little hard to read - is this entire component deprecated?

Reply:
No, just `@Input() schema: any`. I haven't removed it so that I can still compile the app while i make changes to the new inputs: `model`, and `fields`